src/styles/blocks/product.scss

```
@@ -0,0 +1,100 @@
+.product {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 32px 16px 16px;
+  border: 1px solid #e5e5e5;
+  border-radius: 5px;
+  background: $button-color-white;
+  font-size: 12px;
+  line-height: 10px;
+  font-weight: 500;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin-bottom: 4px;
+    margin-top: 0;
+  }
+  
+  &__serial {
+    width: 100%;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $grey-card-color;
+    margin-bottom: 16px;
+    margin-top: 0;
+  }
+
+  &__rewiew,
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
+  }
+
+  &__rewiew {
+    margin-bottom: 24px;
+  
+    &-title {
+      margin: 0;
+      font-weight: 400;
+      line-height: 14px;
+      font-size: 10px;
+      color: $blue-dark-color;
+    }
+  }
+
+  &__price {
+    margin-bottom: 16px;
+  
+    &-title {
+      margin: 0;
+      font-weight: 400;
+      line-height: 18px;
+      color: $grey-card-color;
+    }
+    &-count {
+      margin: 0;
+      font-weight: 700;
+      font-size: 16px;
+      line-height: 18px;
+    }
+  }
+
+  &__btn {
+    align-items: center;
+    background: $blue-light-color;
+    border: 1px solid $blue-light-color;
+    border-radius: 5px;
+    text-decoration: none;
+    color: $button-color-white;
+    padding: 11px 69px;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    text-transform: uppercase;
+
+    &:hover {
+      background: $button-color-white;
+      color: $blue-light-color;
+      transition: all 0.5s;
```

```suggestion
      transition: color 0.5s;
```
you need to explicitly specify the property that is being animated

---

Thanks for the tips!👍